starfish/stack.py

```diff
@@ -825,7 +825,7 @@ def tile_opener(tileset_path, tile, ext):
 
         Writer.write_to_path(
             self._image_partition,
-            filepath,
+            filepath + '.json',
```

Comment:
Hmmm. Maybe check if .json is already in 'filepath' before adding? Or throw an exception if someone is passing in a file instead of a path? I often do .write('blah.json'). So this would make my stuff write to 'blah.json.json' right?